Carsus_atomic_data/Custom_atomic_data.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<p>"STARDUS" typo</p><p> </p><p>Also, "strictly necessarily" is probably supposed to be "necessary." That said, I wouldn't say "necessary" anyway, since you don't technically need a VALD linelist or molecular data from Barklem &amp; Collet. I'd just say that they're recommended.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>